### ISSUE TITLE
Proxy Settings - On/Off slider issue

### ISSUE THREAD

**Issue by ioannis-e** [shouldit/android-proxy#11]
On "Edit wi-fi" screen the "Proxy enabled/disabled" setting can only be set by tapping NOT by sliding.

When sliding to the left for OFF or to the right for ON the setting does not persist and returns to the previous state after a second.

Initially i considered that the app was malfunctioning and was about to uninstall, then I tried tapping and all was well.

Can you please fix?
Thanks.

**Comment by lechuckcaptain** [comment]
You are right, with some devices the App doesn't behave as expected in this view. I'm not sure this is just a matter of tapping instead of switching but I'll definitely try to solve the issue.

Thank you for the feedback!

**Comment by ioannis-e** [comment]
Just for your reference the sliding bug happens on:
- Sony Xpreria Z3 Comapct: Android v4.4.4
- Samsung Galaxy S3: Android v4.3
